.github/workflows/canary-integration-test.yml

```diff
@@ -207,6 +207,11 @@ jobs:
             echo "script failed because wrong realm was passed"
           fi
 
+      - name: test enable v2 mon port
+        run: |
+          toolbox=$(kubectl get pod -l app=rook-ceph-tools -n rook-ceph -o jsonpath='{.items[*].metadata.name}')
+          kubectl -n rook-ceph exec $toolbox -- python3 /etc/ceph/create-external-cluster-resources.py --rbd-data-pool-name replicapool --v2-port-enable
```

Comment:
Can paste the final output of this,
Just for the verification

Reply:
Here's the output with the `v2-port-enable` flag:
```
[
    {
        "name": "rook-ceph-mon-endpoints",
        "kind": "ConfigMap",
        "data": {
            "data": "a=172.31.158.246:3300",
            "maxMonId": "0",
            "mapping": "{}"
        }
    },
    {
        "name": "rook-ceph-mon",
        "kind": "Secret",
        "data": {
            "admin-secret": "admin-secret",
            "fsid": "dcb53605-7ddc-40a9-a77e-1d37f593d03c",
            "mon-secret": "mon-secret"
        }
    },
    {
        "name": "rook-ceph-operator-creds",
        "kind": "Secret",
        "data": {
            "userID": "client.healthchecker",
            "userKey": "AQCOA7lkyWV2ExAASNneNEsLsT9QMsYyqnTBwA=="
        }
    },
    {
        "name": "monitoring-endpoint",
        "kind": "CephCluster",
        "data": {
            "MonitoringEndpoint": "10.132.0.53",
            "MonitoringPort": "9283"
        }
    },
    {
        "name": "rook-csi-rbd-node",
        "kind": "Secret",
        "data": {
            "userID": "csi-rbd-node",
            "userKey": "AQDu9LhkB4tNCRAA0IinHE5/JZjhtFZVi32EOw=="
        }
    },
    {
        "name": "rook-csi-rbd-provisioner",
        "kind": "Secret",
        "data": {
            "userID": "csi-rbd-provisioner",
            "userKey": "AQDt9Lhk9iYLNxAASNU7zG3yePKbgdZYE3xmTw=="
        }
    },
    {
        "name": "rook-ceph-dashboard-link",
        "kind": "Secret",
        "data": {
            "userID": "ceph-dashboard-link",
            "userKey": "https://10.132.0.53:8443/"
        }
    },
    {
        "name": "ceph-rbd",
        "kind": "StorageClass",
        "data": {
            "pool": "replicapool",
            "csi.storage.k8s.io/provisioner-secret-name": "rook-csi-rbd-provisioner",
            "csi.storage.k8s.io/controller-expand-secret-name": "rook-csi-rbd-provisioner",
            "csi.storage.k8s.io/node-stage-secret-name": "rook-csi-rbd-node"
        }
    }
]
```

And, the output without the flag:
```
[
    {
        "name": "rook-ceph-mon-endpoints",
        "kind": "ConfigMap",
        "data": {
            "data": "a=172.31.158.246:6789",
            "maxMonId": "0",
            "mapping": "{}"
        }
    },
    {
        "name": "rook-ceph-mon",
        "kind": "Secret",
        "data": {
            "admin-secret": "admin-secret",
            "fsid": "dcb53605-7ddc-40a9-a77e-1d37f593d03c",
            "mon-secret": "mon-secret"
        }
    },
    {
        "name": "rook-ceph-operator-creds",
        "kind": "Secret",
        "data": {
            "userID": "client.healthchecker",
            "userKey": "AQCOA7lkyWV2ExAASNneNEsLsT9QMsYyqnTBwA=="
        }
    },
    {
        "name": "monitoring-endpoint",
        "kind": "CephCluster",
        "data": {
            "MonitoringEndpoint": "10.132.0.53",
            "MonitoringPort": "9283"
        }
    },
    {
        "name": "rook-csi-rbd-node",
        "kind": "Secret",
        "data": {
            "userID": "csi-rbd-node",
            "userKey": "AQDu9LhkB4tNCRAA0IinHE5/JZjhtFZVi32EOw=="
        }
    },
    {
        "name": "rook-csi-rbd-provisioner",
        "kind": "Secret",
        "data": {
            "userID": "csi-rbd-provisioner",
            "userKey": "AQDt9Lhk9iYLNxAASNU7zG3yePKbgdZYE3xmTw=="
        }
    },
    {
        "name": "rook-ceph-dashboard-link",
        "kind": "Secret",
        "data": {
            "userID": "ceph-dashboard-link",
            "userKey": "https://10.132.0.53:8443/"
        }
    },
    {
        "name": "ceph-rbd",
        "kind": "StorageClass",
        "data": {
            "pool": "replicapool",
            "csi.storage.k8s.io/provisioner-secret-name": "rook-csi-rbd-provisioner",
            "csi.storage.k8s.io/controller-expand-secret-name": "rook-csi-rbd-provisioner",
            "csi.storage.k8s.io/node-stage-secret-name": "rook-csi-rbd-node"
        }
    }
]
```